mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -377,9 +377,9 @@ Specifying a network enables private IP.
 At least `ipv4_enabled` must be enabled or a `private_network` must be configured.
 This setting can be updated, but it cannot be removed after it is set.
 
-* `ssl_mode` - (Optional) Specify how SSL connection should be enforced in DB connections.
+* `ssl_mode` - (Optional) Specify how SSL connection should be enforced in DB connections. Supported values are `ALLOW_UNENCRYPTED_AND_ENCRYPTED`, `ENCRYPTED_ONLY`, and `TRUSTED_CLIENT_CERTIFICATE_REQUIRED` (not supported for SQL Server). See [API reference doc](https://cloud.google.com/sql/docs/postgres/admin-api/rest/v1beta4/instances#ipconfiguration) for details.
```

Comment:
Are any of the values here Beta-only? If not we should link to the GA version of the API documentation to avoid any potential confusion

Reply:
We cannot reintroduce the previous details because the `require_ssl` field has been deprecated in 6.0.0. So we only need to describe the valid values of `ssl_mode`.

I've updated the API version to v1. Thanks.

Reply:
Ah, thanks, I missed that that was a factor!